pkg/nsx/services/securitypolicy/store.go

```diff
@@ -1,153 +1,110 @@
 package securitypolicy
 
 import (
-	"fmt"
-	"strconv"
-	"strings"
-	"sync"
-
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/bindings"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/data"
 	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	"k8s.io/client-go/tools/cache"
 
 	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
-	nsxutil "github.com/vmware-tanzu/nsx-operator/pkg/nsx/util"
-	"github.com/vmware-tanzu/nsx-operator/pkg/util"
 )
 
-func queryTagCondition(service *SecurityPolicyService) string {
-	return fmt.Sprintf("tags.scope:%s AND tags.tag:%s",
-		strings.Replace(util.TagScopeCluster, "/", "\\/", -1),
-		strings.Replace(service.NSXClient.NsxConfig.Cluster, ":", "\\:", -1))
+type resourceAssertion func(i interface{}) interface{}
+
+func keyFunc(obj interface{}) (string, error) {
```

Comment:
Please add some comments to help others to understand the purpose of this important func
such as 
// KeyFunc returns the key of the object used in  cache.Indexers

Reply:
ok